pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java

```diff
@@ -1390,8 +1390,11 @@ public void parallelStagesFailFast() throws Exception {
                         "[Pipeline] { (first)",
                         "{ (Branch: second)",
                         "[Pipeline] { (second)",
-                        "SECOND STAGE FAILED")
-                .logNotContains("Second branch")
+                        "FIRST STAGE FAILURE",
+                        "SECOND STAGE ABORTED")
+                .logNotContains("Second branch",
+                        "FIRST STAGE ABORTED",
+                        "SECOND STAGE FAILURE")
```

Comment:
I added post conditions for these tests to show exactly what conditions fire. Previously, for branch "second", which is aborted because of the failFast setting when branch "first" fails, both the aborted and failure post condition ran. Now, only the aborted condition runs. I did not go back to check what the behavior was _before_ my other PR, but that would probably be a good idea.